doc/manuals/devel/HttpHeaders.md

```diff
@@ -0,0 +1,354 @@
+List of HTTP Headers used by Orion
+----------------------------------------------------------------------------------------------------
+
+1)  Accept                  
+2)  Content-Length          
+3)  Content-Type
+4)  Connection                
+5)  Expect                                                                                
+6)  Host                        	                                                      
+7)  Origin                                                       
+8)  User-Agent              
+9)  X-Forwarded-For                                                               
+10) X-Real-IP
+
+11) Access-Control-Allow-Origin
+12) Access-Control-Allow-Headers                                                                              
+13) Access-Control-Allow-Methods
+14) Access-Control-Max-Age                                 
+15) Access-Control-Expose-Headers
+16) Allow                                                                                    
+17) Location                                                 
+18) Ngsiv2-AttrsFormat                                    
+19) Fiware-Service
+20) Fiware-Servicepath
+21) Fiware-Total-Count
+22) Fiware-Correlator                                     
+
+23) X-Auth-Token
+
+
+---------------------------------------------------------------------------------------------------------------------
+
+# HTTP Headers used by Orion with their Description:
+
+--------------------------------------------------------------------------------------------------------------------
+
+
+## 1. Accept  : 
+
+Accept HTTP Header is used in Request URL. 
+Accept HTTP Header is used with the request URL for accepting the response in well defined format. 
+For example	in request URL if we type:
+	
+		(e.g):      curl ... 'Accept: applicattion/json'
+		 
+On this request we'll get the Response data in well defined Json format.
+
+The API response payloads in this specification are based on `application/json` and  (for attribute value 
+type operation) `text/plain` MIME types. Clients issuing HTTP requests with accept types different 
+than those will get a `406 Not Acceptable` error.
+
+
+
+## 2. Content-Length :
+
+This header is used by both request and response.
+Content-length HTTP header is a mandatory header for the response of the orion.
+it defines the length of the request and response body in octets (8-bit bytes).
+Orion Context Broker expects always a Content-Length header in all client requests, otherwise the client will receive a "411 Length Required" response. 
```

Comment:
Sometimes you use quotes (e.g. "411 Length Required") someimtes you use acent (e.g. `406 Not Acceptable`). Please unify style using always the same.